config/default/common/config/metadata/stories/black_marble_night_lights/step006.md

```diff
@@ -0,0 +1 @@
+Besides being able to see anthropogenic light sources from space, this swirling streak of light over the Hudson Bay is actually an aurora borealis/northern lights event on 4 February 2022.
```

Comment:
We can link to can link to the related Earth Observatory story for more info: https://earthobservatory.nasa.gov/images/149435/brilliant-borealis

Reply:
I had refrained from adding it because it looks so much better in the article, but will add it in.